pageserver/src/tenant/storage_layer/image_layer.rs

```diff
@@ -394,11 +410,19 @@ impl ImageLayerInner {
             }
         }
 
+        // TODO: don't open file twice
```

Comment:
ah: you already have a TODO here for the duplication I just commented on above.  This is worth fixing.

Reply:
https://github.com/neondatabase/neon/pull/6780/commits/0804e0c776b12e841861d53e42f646b5ce92a17d